Makefile

```diff
@@ -23,8 +23,18 @@ CGO_CFLAGS += -DMDBX_DISABLE_VALIDATION=0 # Can disable it on CI by separated PR
 #CGO_CFLAGS += -DMDBX_ENABLE_PROFGC=0 # Disabled by default, but may be useful for performance debugging
 #CGO_CFLAGS += -DMDBX_ENABLE_PGOP_STAT=0 # Disabled by default, but may be useful for performance debugging
 CGO_CFLAGS += -DMDBX_ENV_CHECKPID=0 # Erigon doesn't do fork() syscall
+
+# If it is arm64 or aarch64, then we need to use portable version of blst. use or with stringw "arm64" and "aarch64" to support both
+ifeq ($(shell uname -m), arm64)
+	CGO_CFLAGS += -D__BLST_PORTABLE__
+endif
+ifeq ($(shell uname -m), aarch64)
+	CGO_CFLAGS += -D__BLST_PORTABLE__
+endif
+
+
 CGO_CFLAGS += -D__BLST_PORTABLE__
-CGO_CFLAGS += -Wno-unknown-warning-option -Wno-enum-int-mismatch -Wno-strict-prototypes -Wno-unused-but-set-variable
+CGO_CFLAGS += -Wno-unknown-warning-option -Wno-enum-int-mismatch -Wno-strict-prototypes -Wno-unused-but-set-variable -O3
```

Comment:
usually `go env` has pre-defined O level:
```
CGO_CFLAGS='-O2 -g'
CGO_CXXFLAGS='-O2 -g'
CGO_FFLAGS='-O2 -g'
CGO_LDFLAGS='-O2 -g'
```
not sure how safe to increase it. when i did increase - things did work. but i didn't compare speed.
if you want increase - probably need increase ldflags too.